src/core/main.cpp

```diff
@@ -709,8 +710,7 @@ int main( int argc, char * * argv )
 	ConfigManager::inst()->loadConfigFile(configFile);
 
 	// Hidden settings
-	MixHelpers::setNaNHandler( ConfigManager::inst()->value( "app",
-						"nanhandler", "1" ).toInt() );
+	MixerChannel::setMuteInvalidOutput(ConfigManager::inst()->value("app", "muteinvalidoutput", "1").toInt());
```

Comment:
I believe this needs an upgrade routine to map `nanhandler` -> `<new name TBD>`.

Reply:
I don't see much of an issue making this option a completely new one in the configuration separate from `nanhandler`.

Reply:
> I don't see much of an issue making this option a completely new one in the configuration separate from `nanhandler`.

After some discussion and re-testing, it's effectively the same behavior to the user, no?  I had initially thought that this flag was missing from a stable release, but after some digging, I believe it's been there for a while.  https://github.com/LMMS/lmms/commit/b28d4052404099817abbd080ffc2baa686ee17b4.

I won't fight the upgrade since it's a hidden flag, but I wanted to give you an opportunity to comment before marking this as resolved.  I'll let you make this decision.